packages/litexa-html/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@litexa/html",
-  "version": "0.0.1",
+  "version": "0.4.1",
```

Comment:
should this be 0.4.0?

Reply:
`0.4.1` (current live version of `@litexa/html`) is just the version control catching up to the last `lerna` publish.